src/styles/header-style.ts

```diff
@@ -0,0 +1,22 @@
+export const header_style = {
```

Comment:
variables should be camelCase.

I think you have not setup linters.

@garenpham , can you help please.

Reply:
I think the file should be Pascal Case as well.

HeaderStyle.ts